guide/src/building_and_distribution.md

```diff
@@ -45,20 +45,28 @@ Cross compiling PyO3 modules is relatively straightforward and requires a few pi
 * A toolchain for your target.
 * The appropriate options in your Cargo `.config` for the platform you're targeting and the toolchain you are using.
 * A Python interpreter that's already been compiled for your target.
+* A Python interpreter that is built for your host and available through the `PATH`.
```

Comment:
I think technically it can also be specified using `PYO3_PYTHON` variable?

Reply:
You are absolutely right, but its not documented anywhere. I'll add it in the crosscompiling and in an appropriate section in building

Reply:
Oh it is documented, I'm just blind :sweat_smile: